src/main/java/com/routeme/model/TransitRoute.java

```diff
@@ -10,11 +12,16 @@
 import com.google.maps.model.TravelMode;
 import com.google.maps.model.Vehicle;
 import com.routeme.utility.directions.GoogleDirectionsUtility;
+import com.routeme.utility.directions.RouteParseException;
 
 public class TransitRoute extends Route {
+    boolean firstVehicleInformationSet = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/heshamhamdymassoud/RouteMe-API/file/2638632395/issues/source?bid=3287637&fileBranchId=3320857#l18)